### DISCUSSION TITLE
Workflow not running

### DISCUSSION THREAD

**Discussion by JamesArthurHolland** [community/community#26285] [Actions]
<pre><code class="lang-auto">name: Pull Request Opened / Committed to
# This workflow is triggered on pushes to the repository.
on:
  pull_request:
    branches:
      - '**'
</code></pre>
<p>The above is the predicate I use in our workflow for PRs. It was working previously, but today when I push to the branch of an open PR branch it no longer builds.</p>

**Comment by Zerotask** [comment]
<p>Try <code>on: [push, pull_request]</code></p>

**Comment by JamesArthurHolland** [comment]
<p>I don’t want to build every single branch every time there is a commit.</p>
<p>We only rebuild on pushes to branches for which there is an open pull request.</p>
<p>This functionality was working previously, we’ve been doing it this way for months. Something has broken recently.</p>
<p>I think that pull_request is no longer triggered for pushes to the branch</p>

**Comment by JamesArthurHolland** [comment, marked as answer]
<p>OK. I see the problem. If the repo can’t automatically merge, then it fails silently. No workflow runs.</p>
<p>Fixed the merge conflict in the submodule and it works now.</p>

**Comment by the-matt-morris** [comment]
<p>Thank you.  I had a similar problem and was banging my head against the wall until I saw your post.</p>
